docs/dg/devOpsGuide.md

```diff
@@ -28,9 +28,9 @@ This page documents the various components that form part of the DevOps infrastr
 
 All three types of tests in the test suite are run in a single GitHub Actions workflow called "Continuous Integration". The steps are defined in [`integration.yml`](https://github.com/reposense/RepoSense/blob/master/.github/workflows/integration.yml) and are split into four types of jobs:
 
-1. Ubuntu JDK 8 (`ubuntu`): Runs both unit tests and system tests on JDK 1.8 running on supported Ubuntu versions. This job also produces a RepoSense report and the MarkBind documentation website for previewing.
-2. macOS JDK 8 (`macos`): Runs both unit tests and system tests on JDK 1.8 running on supported macOS versions.
-3. Windows JDK 8 (`windows`): Runs both unit tests and system tests on JDK 1.8 running on supported Windows versions.
+1. Ubuntu JDK 8 (`mainbuild`): Runs both unit tests and system tests on JDK 1.8 running on supported Ubuntu versions. This job also produces a RepoSense report and the MarkBind documentation website for previewing.
+2. macOS JDK 8 (`mainbuild`): Runs both unit tests and system tests on JDK 1.8 running on supported macOS versions.
+3. Windows JDK 8 (`mainbuild`): Runs both unit tests and system tests on JDK 1.8 running on supported Windows versions.
```

Comment:
I wonder if these can simply be compressed to state that `mainbuild` runs unit and system tests on 6 different OS, with codecov being run on which 3 OS and additional stuff like report and documentation specifically on 20.04. Or perhaps we can even leave out the specific OS mentioned since those might get outdated and give a more general statement.

Reply:
True, maybe just say:
- Runs unit and system tests on supported versions of Ubuntu, MacOS and Windows. Ubuntu is also used for generating the RepoSense report and MarkBind documentation website for previewing.

Reply:
sounds good, maybe remove the brackets around Ubuntu/MacOS/Windows. Besides that, I wonder if it's worth mentioning about running the codecov here as well, unless it is documented elsewhere.

Reply:
Codecov is somewhere below in the same doc